api.md

```diff
@@ -132,23 +148,68 @@ require('s3scan').Purge('s3://my-bucket/my-key', function(err) {
 });
 ```
 
-Returns **[object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)** a writable stream
+Returns **[object][19]** a writable stream
 
 ## s3scan.Copy
 
 Provides a writable stream that accepts keys and copies them to another location.
 
-**Parameters**
+### Parameters
 
--   `fromBucket` **[string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)** the bucket to copy objects from
--   `toBucket` **[string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)** the bucket to copy objects into
--   `keyTransform` **[function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function)?** a function to transform keys. The
+*   `fromBucket` **[string][18]** the bucket to copy objects from
+*   `toBucket` **[string][18]** the bucket to copy objects into
+*   `keyTransform` **[function][22]?** a function to transform keys. The
     function you provide should accept a source key and synchronously return the
     desired destination key. If not provided, objects in the `fromBucket` will be
     copied to the `toBucket` as-is.
--   `options` **[object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)?** options to provide to the writable stream.
-    -   `options.agent` **[object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)?** an HTTPS agent to use for S3 requests
-    -   `options.s3` **[object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)?** an S3 client to use to make requests
-    -   `options.concurrency` **[number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)?** concurrency at which to copy objects
+*   `options` **[object][19]?** options to provide to the writable stream.
+
+    *   `options.agent` **[object][19]?** an HTTPS agent to use for S3 requests
+    *   `options.s3` **[object][19]?** an S3 client to use to make requests
+    *   `options.concurrency` **[number][21]?** concurrency at which to copy objects
+
+Returns **[object][19]** a writable stream
+
+[1]: #s3scanlist
```

Comment:
Why are all the links split into this labels section? Why use numbers for the labels instead of clear text? E.g. instead of `[20]`, a more readable label would be `[boolean]`.

Reply:
This is autogenerated with the documentation dependency. `"docs": "documentation build -f md > api.md"`